doc/install/index.rst

```diff
@@ -1,17 +1,23 @@
 DAGMC installation overview
 ===========================
 
-This document describes the DAGMC installation process. Before you begin, you
-should know that the following will be required:
+This document describes the DAGMC installation process. 
+
+If you plan to use OpenMC_, you can follow `their installation 
+instructions <OpenMC_install>`_ with
+DAGMC built-in.
+
+Otherwise.... Before you begin, you should know that the following will be
+required:
 
 1.  A basic understanding of Unix/Linux shell commands and how to navigate
     the shell, for installing dependencies and DAGMC enabled codes
-2.  MCNP5_ or MCNP6_ source code, if you intend to install DAG-MCNP5 or
-    DAG-MCNP6
+2.  MCNP6_ source code, if you intend to install DAG-MCNP6 (or MCNP5_ source
+    code for DAG-MCNP5)
 3.  FLUKA_, if you intend to install FluDAG
 4.  Coreform Cubit_ for the creation of geometry
```

Comment:
Keen to see other tools mentioned for geometry creation but I might be a bit bias :-)

perhaps throw in a link to https://github.com/svalinn/DAGMC/discussions/812

Reply:
I agree @shimwell , but think we need to develop a better strategy than just a link to #812 - feel free to PR some doc updates that address this.

Reply:
Thanks for being open to this, Once I've got that [cad-to-dagmc ](https://github.com/fusion-energy/cad_to_dagmc) passing a decent range of geometry comparisons over on the [benchmark zoo](https://github.com/fusion-energy/model_benchmark_zoo) then I shall be back with a PR to add a code to the list